go/src/koding/db/models/groupdata.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+// GroupData holds a group's private info.
+type GroupData struct {
+	ID   string `json:"_id"`
+	Slug string `json:"slug"`
+	Data *Data  `bson:"data,omitempty" json:"data,omitempty"`
```

Comment:
I was under the impression that [Bar.Data](https://play.golang.org/p/S2GAI9Xp2w) wouldnt be nil and that was my main driving force to use pointer.
Also:
`when in doubt, use a pointer receiver.`